data/genesis/decaf.toml

```diff
@@ -16,3 +18,17 @@ timestamp = "1970-01-01T00:00:00Z"
 
 [l1_finalized]
 number = 6758037 # Block where fee contract is deployed
+
+[[upgrade]]
+version = "0.3"
+start_proposing_view = 0
+stop_proposing_view = 4000000
+
+[upgrade.epoch]
+[upgrade.epoch.chain_config]
+chain_id = 0xdecaf
```

Comment:
Is this the same as the current chain id?

Reply:
Yes